### PR TITLE
Entities

### PR DESCRIPTION
**Pull request by MartynasKasp**

### CODE REVIEW

File: server/server/Models/BugReport.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace server.Models
+{
+    public class BugReport
+    {
+        public int Id { get; set; }
+        public string Contnet { get; set; }
```
Comment (MantasGra) [inline server/server/Models/BugReport.cs]: nice variable name

**Review by MantasGra — COMMENTED** [review]
Looks good, reiktų kad Mykolas arba Povilas pažiūrėtų tho

File: server/server/Models/BugReport.cs (hunk 1 above)
Comment (MartynasKasp) [reply server/server/Models/BugReport.cs]: fixed

**Review by pockeviciuspovilas — CHANGES_REQUESTED** [review]
Need annotations [Required] to fix non-nullable types, also use annotations like [Column(TypeName = "varchar(xxx)")] to optimize DB a bit.

**Review by MartynasKasp — COMMENTED** [review]
Changes were made

**Review by MantasGra — APPROVED** [review]
Solid work

**Review by pockeviciuspovilas — APPROVED** [review]
Looks good, well done

**Pull request merged**